src/redux/time-entries/selectors.js

```diff
@@ -10,22 +10,24 @@ export const timeEntriesRootSelector = createSelector(
 export const timeEntriesItemsSelector = createSelector(
   timeEntriesRootSelector,
   clientsItemsSelector,
-  (timeEntries, clients) =>
-    timeEntries.items
+  (timeEntries, clients) => {
+    return timeEntries.items
```

Comment:
Why do you explicitly return this?

Reply:
Ah this was still a remnant from debugging/console-logging.. will rewrite!